modules/javascript/partytown-web-worker/can-load.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Can load function to determine if Partytown Web Worker module can be loaded.
+ *
+ * @since   n.e.x.t
+ * @package partytown-web-worker
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! function_exists( 'perflab_partytown_web_worker_can_load' ) ) {
```

Comment:
Why are these pluggable functions?

Reply:
Fixed in https://github.com/WordPress/performance/pull/556/commits/ac2c9d0d5b900969699c35979a5b4afabb2aae97